bindings/c/Test.cpp

```diff
@@ -203,8 +203,8 @@ TEST_CASE_METHOD(Test, "Parameter-framework c api use") {
         }
         GIVEN("A criteria with lots of values")
         {
-            // Build a criterion with as many value as there is bits in int.
-            std::vector<char> names(sizeof(int) * CHAR_BIT + 1, 'a');
+            // Build a criterion with a lot of value
+            std::vector<char> names(300, 'a');
```

Comment:
Why 300 ?  I think this test section is meaningless now (was testing the criterion value number limitation). Delete it ?